src/test/datascience/notebook/helper.ts

```diff
@@ -88,7 +93,7 @@ export async function insertMarkdownCell(source: string, options?: { index?: num
     }
     const startNumber = options?.index ?? activeEditor.document.cellCount;
     await chainWithPendingUpdates(activeEditor.document, (edit) =>
-        edit.replaceNotebookCells(activeEditor.document.uri, new NotebookRange(startNumber, 0), [
+        edit.replaceNotebookCells(activeEditor.document.uri, new NotebookRange(startNumber, startNumber), [
```

Comment:
Peng helped convert these over for us, but we didn't realize that the old API used to take start, 0 to mean start and 0 more. Converted to fix tests.